edge/cmd/edgemark/hollow_edgecore.go

```diff
@@ -143,7 +143,11 @@ func EdgeCoreConfig(config *hollowEdgeNodeConfig) *v1alpha2.EdgeCoreConfig {
 
 	// overWrite config
 	edgeCoreConfig.EdgeCoreVersion = version.Get().String()
-	edgeCoreConfig.DataBase.DataSource = "/edgecore.db"
```

Comment:
Edgemark is a tool used for simulating edge nodes in scalability tests. Its characteristics lie in rapid replication and quick startup, so it does not involve much customization.  Therefore, I think there may be no need to consider configurable datapaths for now. If such a requirement arises in the future, it can be added as a parameter of hollowEdgeNodeConfig. : )